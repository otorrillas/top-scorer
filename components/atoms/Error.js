import React from 'react';
import PropTypes from 'prop-types';

const Error = ({ message }) => (
  <>
    <h1>Error</h1>
    <p>{message}</p>
  </>
);

Error.propTypes = {
  message: PropTypes.string.isRequired
};

export default Error;
