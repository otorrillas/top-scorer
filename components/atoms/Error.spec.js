import React from 'react';
import renderer from 'react-test-renderer';

import Error from './Error';

describe('<Error />', () => {
  const renderComponent = props => renderer.create(<Error {...props} />);

  describe('@render', () => {
    it('default', () => {
      const tree = renderComponent({
        message: 'An unexpected error'
      });
      expect(tree).toMatchSnapshot();
    });
  });
});
