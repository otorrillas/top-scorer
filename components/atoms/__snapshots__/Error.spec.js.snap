// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> @render default 1`] = `
Array [
  <h1>
    Error
  </h1>,
  <p>
    An unexpected error
  </p>,
]
`;
